### DISCUSSION TITLE
Welcome to web-components Discussions!

### DISCUSSION THREAD

**Discussion by jolanglinais** [accordproject/web-components#221] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hemant767** [comment]
I am not able to run this in local system .
After 
"npm run storybook" It's showing error

**Reply by jolanglinais** [reply]
Sorry to hear it! You could open an issue and document the steps you took, so we can help you out.
